environments/.env.production

```diff
@@ -23,6 +23,7 @@ SENDWYRE_API_URL=https://api.sendwyre.com
 SENDWYRE_WIDGET_URL=https://pay.sendwyre.com/purchase
 SENDWYRE_ACCOUNT_ID=AC_ERP9DMNTAMB
 SENDWYRE_ENVIRONMENT=prod
+SENDWYRE_RETURN_URL=https://offers-webapp-qa.nonprod.pillarproject.io/sendwyre
```

Comment:
Do we have a prod link?

Reply:
We don't, actually. I'm asking for it and will update.